google/auth/_service_account_info.py

```diff
@@ -21,6 +21,17 @@
 
 from google.auth import crypt
 
+try:
+    from google.auth import _cryptography_impl
+except ImportError:
+    _cryptography_impl = None
+
+
+if _cryptography_impl is None:
+    DefaultSigner = crypt.RSASigner
+else:
+    DefaultSigner = _cryptography_impl.CryptographySigner
```

Comment:
@jonparrott I wasn't sure if you'd want me to

- define `DefaultSigner` at all
- define `DefaultSigner` here or in the `ImportError` blocks